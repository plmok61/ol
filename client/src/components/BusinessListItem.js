import React from 'react'

const BusinessListItem = ({ business }) => {

  const viewBusiness = () => {
    window.location = `/#/businesses/${business.id}`
  } 

  return (
    <div onClick={viewBusiness}>
      {business.name}
    </div>
  )
}

export default BusinessListItem