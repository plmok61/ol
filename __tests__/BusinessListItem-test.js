import React from 'react'
import BusinessListItem from '../client/src/components/BusinessListItem'
import renderer from 'react-test-renderer'

test('BusinessListItem renders', () => {
  const component = renderer.create(
    <BusinessListItem business={{name: 'Test Company'}}/>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})