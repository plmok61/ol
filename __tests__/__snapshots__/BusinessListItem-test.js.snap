exports[`test BusinessListItem renders 1`] = `
<div
  className="business-list-item"
  onClick={[Function]}>
  Test Company
</div>
`;

exports[`test test 3 1`] = `
<div
  className="business-list-item"
  onClick={[Function]}>
  Test Company
</div>
`;
